### PR TITLE
sys-process/gotop: new package

### PR DESCRIPTION
**Pull request by tormath1**
Hi! 

Gotop is another terminal based graphical activity monitor, inspired by gtop and vtop, this time written in Go.

I added `-mod` option to fetch dependencies from `vendor/`. I was not able to fetch dependencies with only `GO111MODULE=on` for some obscure reasons.

Let me know for the changes, this is my first PR for a Gentoo ebuild.

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
have you checked [this](https://gitlab.com/gekola/chaoslab-overlay/blob/master/sys-process/gotop/gotop-3.0.0.ebuild)?

**Comment by tormath1** [comment]
Hi @Alessandro-Barbieri , I was not aware of this ! Does it mean we can close this PR and let user use chaoslab-overlay if they want to install Gotop ?

File: sys-process/gotop/gotop-3.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+inherit golang-vcs-snapshot
+
+EGO_PN="github.com/cjbassi/${PN}"
+
+DESCRIPTION="A terminal based graphical activity monitor inspired by gtop and vtop"
+HOMEPAGE="https://github.com/cjbassi/gotop"
+
+SRC_URI="https://github.com/cjbassi/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="AGPL-3"
```
Comment (mgorny) [inline sys-process/gotop/gotop-3.0.0.ebuild]: Please list LICENSEs for all bundled libraries. You can use `dev-util/ninka` to automatically check what particular LICENSE file is.

**Comment by Alessandro-Barbieri** [comment]
@tormath1 here some random thoughts
* ask the author of the other ebuild to have it added to gentoo
* take the other ebuild, review/fix/wathever it and maintain it here
* go on with your ebuild
* close the PR and use the overlay

Personally I don't like using overlays

**Comment by Alessandro-Barbieri** [comment]
Are you still interested in packaging this?
The package in the other overlay fail to build and I would like to have gotop in guru.

**Comment by tormath1** [comment]
In this case I could rework on it this weekend or you can take the lead of this topic. 😉

**Comment by tormath1** [comment]
I close this PR, since this package will live on guru (https://github.com/gentoo/guru/commit/7556ef0da625367b40d5b0ce7cbd92141a3e5a81)

**Pull request closed without merging**